docs/advanced/_category_.json

```diff
@@ -1,6 +1,6 @@
 {
   "label": "Advanced Configuration",
-  "position": 5,
+  "position": 6,
```

Comment:
Why adjust the position of `Advanced Configuration`?

Reply:
Done